tasks/data_handling/tests/test_data_pre_processing.py

```diff
@@ -0,0 +1,283 @@
+import scanpy as sc
+import unittest
+import os
+import numpy as np
+from collections import Counter
+
+import global_settings as gs
+from tasks.data_handling import data_pre_processing
+from tasks.test_helper.builders.data_handling import data_pre_process as test_data_pre_process
+from tasks.test_helper.builders.utils import context_utils as cu
+
+
+class TestCreateCountMatrix(unittest.TestCase):
+    def test_need_file_path(self):
+        with self.assertRaises(ValueError) as context:
+            result = data_pre_processing.create_count_matrix()
+
+    data_path = os.path.join('\\'.join(os.getcwd().split('\\')[:-2]), gs.TEST_DATA_PATH)
+
+    def tearDown(self) -> None:
+        cu.clean(folder_path=self.data_path)
+
+    file_name = 'simulated_csc_h5_data.h5'
+    simulated_matrix = test_data_pre_process.simulate_csc_h5_data(file_path=data_path,
```

Comment:
Same for these class variables. I was confused by the `self.simulated_matrix` because I hadn't seen that these are actually class variables and not inside a function. Always on before any function